test/unit/configuration/configuration_parser_test.js

```diff
@@ -0,0 +1,139 @@
+const assert = require('assert');
+const configurationParser = require('./../../../lib/configuration/configuration_parser');
+const fsPromises = require('fs/promises');
+const os = require('os');
+const path = require('path');
+const { v4: uuidv4 } = require('uuid');
+const tempDir = path.join(os.tmpdir(), uuidv4());
```

Comment:
you can create temporary dirs using build in `fs.mkdtemp` and don't use `os` and `uuid` dependencies here

Reply:
changed to mkdtemp. I got rid of uuid, but still I need os, to get tempdir.